q4.js

```diff
@@ -0,0 +1,37 @@
+let sa = function SortArray(inputArray){
```

Comment:
SortArray was supposed to be a constructor function.

Reply:
Made the changes